Documentation/HowTo/adding-disqus-comments-to-posts.md

```diff
@@ -0,0 +1,44 @@
+# How to add Disqus comments to posts
+
+## Getting Short Name from Disqus
+
+Before installing, make sure you've [registered](https://disqus.com/register/) a Disqus [shortname](https://help.disqus.com/customer/portal/articles/286833), and this will be used to reference all of your comments and settings.
+
+
+Once you have got your short name, you can proceed to the next step
+
+## JS
+
+Create a new file called `disqus.js` in your assets folder and paste this code
+
+```javascript
+(function() {  // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
```

Comment:
I think this code sample would be a lot easier to understand if it didn't use cryptic variable names like `d` and `s`.